python/mpt/tests/src/mpt/test_utils.py

```diff
@@ -15,6 +15,13 @@
 from mpt.utils import check_branch_node, nibble_list_to_bytes
 from tests.utils.args_gen import AddressAccountNodeDiffEntry
 
+list_address_account_node_diff_entry_strategy = st.lists(
+    st.from_type(AddressAccountNodeDiffEntry),
+    min_size=1,
```

Comment:
```suggestion
    min_size=0,
```

any reason to use a non-empty list?

Reply:
the patched hints will not raise

Reply:
then you should not be using this strategy in the happy path test